content/en/docs/concepts/policy/limit-range.md

```diff
@@ -373,6 +373,11 @@ Delete the `limitrange-demo` namespace to free all resources:
 ```shell
 kubectl delete ns limitrange-demo
 ```
+Change your context to `default` namespace with the following command:
```

Comment:
Maybe add a Clean up header for this?

Reply:
Added clean up header as per suggestion.